kong/plugins/hmac-auth/access.lua

```diff
@@ -2,22 +2,25 @@ local utils = require "kong.tools.utils"
 local responses = require "kong.tools.responses"
 local constants = require "kong.constants"
 local singletons = require "kong.singletons"
+local crypto = require "crypto"
 local resty_sha256 = require "resty.sha256"
+local new_table = require "table.new"
```

Comment:
we typically wrap this in pcall. see https://github.com/Mashape/kong/blob/master/kong/plugins/ip-restriction/handler.lua#L5 as an example